README.md

```diff
@@ -489,3 +490,10 @@ They are as follows:
 [installodh]: https://opendatahub.io/docs/quick-installation
 [kfp-tekton]: https://github.com/kubeflow/kfp-tekton
 [kfp]: https://github.com/kubeflow/pipelines
+
+# Configuring Log Levels for the Operator
+
+By default, the operator's log messages are set to `info` severity. 
+If you wish to adjust the log verbosity, you can do so by modifying the `--zap-log-level=info` arg within the `spec.template.spec.containers[0].args` section of the [manager.yaml](config/manager/manager.yaml) file to your preferred severity level.
+
+For a comprehensive list of available values, please consult the [Zap documentation](https://pkg.go.dev/go.uber.org/zap#pkg-constants).
```

Comment:
Can you move this above the references to the links

Reply:
Done.